outfit.js

```diff
@@ -0,0 +1,14 @@
+class Outfit{
+  constructor(){
+    this.hats = hats
+    this.clothes = clothes
+    this.accessories = accesories
+  }
+  addGarment(){
+
+  }
+
+  removeGarment(){
+
+  }
+}
```

Comment:
Thank for working on this part of Iteration 1.  This will help us save time when we start on future iteration of the project.